src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubRouteCommand.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Commands.Network
     using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
     using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
-    [CmdletDeprecation(ReplacementCmdletName = "Add-AzVirtualHubRoute")]
+    [CmdletDeprecation(ReplacementCmdletName = "New-AzVHubRoute")]
```

Comment:
Why did the command name change here

Reply:
initially we marked this deprecated and pointed it to use rtv2, so now replaced it to point to rtv3